src/tilemap/TileLayer.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Threading.Tasks.Dataflow;
+
+namespace Tile_Mapper.tilemap
+{
+    //Class to store and manipulate information related to tile layers
```

Comment:
Make sure to follow the C# guidelines. In this case, your comment will be `// Class to store and manipulate information related to tile layers.`